lenses_python/Policy.py

```diff
@@ -0,0 +1,33 @@
+from requests import *
+
+class Policy:
+    def __init__(self, token, url):
+        """
+
+        :param token:
+        """
+        self.token = token
+        self.url = url
+        self.url_extend = "/api/protection/policy"
+        self.default_headers = {'Content-Type': 'application/json', 'Accept': 'application/json',
+                                'x-kafka-lenses-token': self.token}
+
+    def view_policy(self):
+        """
+        View all versions for a policy
+
+        :return:
+        """
+        url = self.url+self.url_extend
```

Comment:
better to use `string interpolation` with `f` as python 3 supports it

Reply:
ok

Reply:
This fails with 3.5